### ISSUE TITLE
libcap 2.69 and/or bubblewrap broken

### ISSUE THREAD

**Issue by joemiller** [wolfi-dev/os#11139]
This is a recent issue, so my guess is it started around the time of the libcap 2.69 epoch 1 bump: https://github.com/wolfi-dev/os/commit/27f16890251fcd9c721be7cee58b1ee52f6e7f6f

steps to reproduce:

```console
$  docker run --rm -it --pull=always cgr.dev/chainguard/wolfi-base        

7628418f9f15:/# apk add bubblewrap
fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
ERROR: unable to select packages:
  libcap-2.68-r0:
    conflicts: libcap-2.69-r1 libcap-2.69-r1[so:libcap.so.2=2]
    satisfies: libcap-2.69-r1[so:libpsx.so.2] bubblewrap-0.8.0-r0[so:libcap.so.2]
  libcap-2.69-r1:
    conflicts: libcap-2.68-r0 libcap-2.68-r0[so:libcap.so.2=2]
    satisfies: bubblewrap-0.8.0-r0[so:libcap.so.2]
```

pinning `libcap==2.68-r0` allows for `bubblewrap` to be installed successfully:

```console
7628418f9f15:/# apk add libcap==2.68-r0 bubblewrap
(1/2) Installing libcap (2.68-r0)
(2/2) Installing bubblewrap (0.8.0-r0)
OK: 15 MiB in 16 packages
```

maybe bubblewrap also needs an r1 bump?

**Comment by debasishbsws** [comment]
https://github.com/wolfi-dev/os/pull/11056

**Comment by jonjohnsonjr** [comment]
This should be fixed in libcap-2.69-r2.apk as of https://github.com/wolfi-dev/os/pull/11247
